code/src/pages/Error.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+export const Error = () => {
+    return (
+        <div className="error">
+            <h1>ERROR page</h1>
+            <p>The Movie ID is not valid</p>
+        </div>
+    )
+}
```

Comment:
If I end up on this page, wouldn't it be nice with a back button or something that takes me back to the main page?

Reply:
That is a great idea. I didn't think about that :D